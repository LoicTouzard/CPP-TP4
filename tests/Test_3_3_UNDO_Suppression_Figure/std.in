C c1 0 0 1
DELETE c1
UNDO
LIST
EXIT
