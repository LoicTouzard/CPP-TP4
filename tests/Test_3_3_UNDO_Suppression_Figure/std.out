OK
#New object :c1
OK
OK
C c1 0 0 1
